app/models/volunteer.rb

```diff
@@ -60,21 +62,21 @@ def supervised_by?(supervisor)
     self.supervisor == supervisor
   end
 
-  def made_contact_with_all_cases_in_days?(num_days = 14)
+  # false if volunteer has any case with no contact in the past 30 days
+  def made_contact_with_all_cases_in_days?(num_days = CONTACT_MADE_IN_DAYS_NUM)
     # TODO this should do the same thing as no_contact_for_two_weeks but for a volunteer
-    total_cases_count = casa_cases.count
+    total_cases_count = casa_cases.size
```

Comment:
interesting, why size vs count? is one of them more database-y?

Reply:
@compwron When `casa_cases` have already been loaded, `size` pulls the length of the relation, whereas `count` issues another DB query.